components/dot_menu/dot_menu.test.tsx

```diff
@@ -95,20 +96,20 @@ describe('components/dot_menu/DotMenu', () => {
     test('should not have divider when able to edit or delete a system message', () => {
         const props = {
             ...baseProps,
-            post: {
+            post: TestHelper.getPostMock({
                 ...baseProps.post,
-                type: PostTypes.JOIN_CHANNEL,
-            },
+                type: 'system_join_channel',
```

Comment:
You could continue using the constant with `type: PostTypes.JOIN_CHANNEL as PostType,`

Reply:
done in latest commit